src/store/model-store.js

```diff
@@ -1,33 +1,26 @@
 import { Store } from 'consus-core/flux';
-
+import { searchModel } from '../lib/api-client';
 let model = null;
 let models = [];
 
 class ModelStore extends Store {
 
     getModel() {
-        if (model === null) {
-            return model;
-        }
-        return {
-            address: model.address,
-            name: model.name
-        };
+        return model;
     }
 
     getAllModels(){
         return models;
     }
-
+    searchModelByAddress(address){
```

Comment:
It was because i was using the stores to call the api, I need to move the two occurrences of this in item-store and here in model-store